src/test/java/guitests/PanelRenameTest.java

```diff
@@ -80,121 +77,32 @@ public void panelRenameTest() {
-        FilterPanel panel5 = (FilterPanel) panels.getPanel(5);
-        Text panelNameText5 = panel5.getNameText();
-        assertEquals("Panel", panelNameText5.getText());
-        sleep(EVENT_DELAY);
-        
-        pushKeys(CREATE_RIGHT_PANEL);
-        
-        // Testing typing more characters when textfield is full
-        // Expected: new name accepted with additional characters not added
-        PlatformEx.runAndWait(() -> UI.events.triggerEvent(new ShowRenamePanelEvent(6)));
-        sleep(EVENT_DELAY);
-        String randomName6 = RandomStringUtils.randomAlphanumeric(PANEL_MAX_NAME_LENGTH);
-        PanelNameTextField renameTextField6 = find("#dummy/dummy_col6_renameTextField");
-        renameTextField6.setText(randomName6);
-        
-        int randomCaret6 = rand.nextInt(PANEL_MAX_NAME_LENGTH - 1);
-        renameTextField6.positionCaret(randomCaret6);
-        type("characters that will not be added");
-        push(KeyCode.ENTER);
-        FilterPanel panel6 = (FilterPanel) panels.getPanel(6);
-        Text panelNameText6 = panel6.getNameText();
-        assertEquals(randomName6, panelNameText6.getText());
-        sleep(EVENT_DELAY);
-        
-        pushKeys(CREATE_RIGHT_PANEL);
-        
-        PlatformEx.runAndWait(() -> UI.events.triggerEvent(new ShowRenamePanelEvent(7)));
-        sleep(EVENT_DELAY);
-        String randomName7 = RandomStringUtils.randomAlphanumeric(PANEL_MAX_NAME_LENGTH - 4);
-        PanelNameTextField renameTextField7 = find("#dummy/dummy_col7_renameTextField");
-        renameTextField7.setText(randomName7);
-        
-        int randomCaret7 = rand.nextInt(randomName7.length());
-        renameTextField7.positionCaret(randomCaret7);
-        // Random string that will make the name longer than max length
-        String randomString = RandomStringUtils.randomAlphanumeric(5);
-        type(randomString);
-        push(KeyCode.ENTER);
-        String expected = (randomName7.substring(0, randomCaret7) + 
-                randomString.substring(0, 4) + randomName7.substring(randomCaret7, randomName7.length()));
-        FilterPanel panel7 = (FilterPanel) panels.getPanel(7);
-        Text panelNameText7 = panel7.getNameText();
-        assertEquals(expected, panelNameText7.getText());
-        sleep(EVENT_DELAY);
 
         // Testing whether the close button appears once rename box is opened.
         // Expected: Close button should not appear once rename box is opened and while edits are being made.
         //           It should appear once the rename box is closed and the edits are done.
-        pushKeys(CREATE_RIGHT_PANEL);
-        waitUntilNodeAppears("#dummy/dummy_col8_closeButton");
-        boolean isPresentBeforeEdit = exists("#dummy/dummy_col8_closeButton");
-        PlatformEx.runAndWait(() -> UI.events.triggerEvent(new ShowRenamePanelEvent(8)));
+        press(CREATE_RIGHT_PANEL);
+        boolean isPresentBeforeEdit = exists("#dummy/dummy_col3_closeButton");
```

Comment:
We most likely need the wait. It was added during the meeting just now.

Also pushKeys was used because press causes some problems with the CI (#1047).